### DISCUSSION TITLE
Animated Circular Progress Bar - Tailwind colors

### DISCUSSION THREAD

**Discussion by JasonColeyNZ** [magicuidesign/magicui#400] [Q&A]
Hi there,
Can the Animated Circular Progress Bar use tailwind colors in the gaugePrimaryColor and guageSecondaryColor properties?

Thanks

**Comment by Yeom-JinHo** [comment]
@JasonColeyNZ 

Yes. You can use Tailwind theme colors by passing the CSS variables as strings:
```jsx
<AnimatedCircularProgressBar
  value={value}
  gaugePrimaryColor="var(--color-primary)"
  gaugeSecondaryColor="var(--color-muted)"/>
```
stroke accepts any valid CSS color, including var(--color-primary), var(--primary), var(--muted), etc., so these will work with your theme and dark mode.
